config/i18n.json

```diff
@@ -816,6 +816,7 @@
     "ColumnSize": "{{num}} items",
     "ColumnSizeAuto": "Auto",
     "CsvImport": "Import tags/notes from CSV",
+    "CustomStatDesc": "Choose armor stats to make a custom total stat that will show on the item popup, Organizer, and Compare. Search with stat:custom:>=30.",
```

Comment:
I think that's fair. I just don't know how to effectively communicate with folks that they can now search using this.

Reply:
we can mess with it when filters are self contained and they can generate their own help blurbs. this could link to a stat help modal for just `stat:`